scripts/build-push-all.sh

```diff
@@ -6,6 +6,7 @@ HELM_CHART_VERSION=v4
 SUMO_REGISTRY="public.ecr.aws/sumologic/"
 PUSH=""
 CHECK="${CHECK:-true}"
+OPERATOR_PROJECT_ID=6075d88c2b962feb86bea730
 
```

Comment:
why do we have a default here and what is it?

Reply:
Maybe add a comment here that explains it.

Reply:
It's static :D
I will add `readonly`

Reply:
done :)